src/components/Creation/Overview.js

```diff
@@ -12,149 +12,150 @@ const defaultOpt = "--";
 
 export default class Overview extends Component {
 
-    constructor(props) {
-        super(props)
-        var data = this.props.data;
-        this.state = {
-            jobTitle: data.jobTitle || "",
-            company: data.company || "",
-            salary: data.salary || "",
-            travel: data.travel || "",
-            worklife: data.worklife || "",
-            physical: data.physical || "",
-            workenv: data.workenv || "",
-            isValid: data.isValid || false
-        }
-        this.rules = {
-            jobTitle: { required: true },
-            company: { required: true },
-            salary: { numeric: true },
-            travel: { required: true },
-            worklife: { required: true },
-            physical: { required: true },
-            workenv: { required: true }
-        }
-    }
+  constructor(props) {
+      super(props)
+      var data = this.props.data;
+      this.state = {
+        jobTitle: data.jobTitle || "",
+        company: data.company || "",
+        salary: data.salary || "",
+        travel: data.travel || "",
+        worklife: data.worklife || "",
+        physical: data.physical || "",
+        workenv: data.workenv || "",
+        isValid: data.isValid || false
+      }
+      this.rules = {
+        jobTitle: { required: true },
+        company: { required: true },
+        salary: { numeric: true },
+        travel: { required: true },
+        worklife: { required: true },
+        physical: { required: true },
+        workenv: { required: true }
+      }
+  }
 
-    updateFormValue = (event) => {
-        if(event.target.value == '--') event.target.value = null;
-            this.setState({ [event.target.name]: event.target.value },
-                () => this.props.update(this.state))
-    }
+  updateFormValue = (event) => {
+    const { name, value } = event.target; 
+    if(value == '--') value = null;
```

Comment:
Seeing error here when selecting a value (say 10%) and then selecting '--' after for the dropdowns on overview section

![image](https://user-images.githubusercontent.com/41658758/114322871-7cb50a00-9af0-11eb-8345-d58a1bb5634d.png)